grpc/nodejs/BUILD

```diff
@@ -41,30 +41,30 @@ nodejs_binary(
 ts_grpc_compile(
     name = "protocol",
     deps = [
-        "//protobuf:answer-proto",
-        "//protobuf:concept-proto",
-        "//protobuf:database-proto",
-        "//protobuf:grakn-proto",
-        "//protobuf:logic-proto",
-        "//protobuf:options-proto",
-        "//protobuf:query-proto",
-        "//protobuf:session-proto",
-        "//protobuf:transaction-proto",
+        "//common:answer-proto",
+        "//common:concept-proto",
+        "//common:logic-proto",
+        "//common:options-proto",
+        "//common:query-proto",
+        "//common:session-proto",
+        "//common:transaction-proto",
+        "//core:database-proto",
+        "//core:service-proto",
     ],
     grpc_deps = [
-        "//protobuf:grakn-proto",
+        "//core:service-proto",
     ]
 )
 
 ts_grpc_compile(
     name = "protocol-cluster",
     deps = [
-        "//protobuf/cluster:cluster-proto",
-        "//protobuf/cluster:database-proto",
-        "//protobuf/cluster:grakn-cluster-proto",
+        "//cluster:server-proto",
+        "//cluster:database-proto",
+        "//cluster:service-proto", # TODO: do we need this?
```

Comment:
Let's remove it since it's already declared in `grpc_deps`

Reply:
I don't know if that will work - if we do remove it, we should re-test it.

Reply:
I'm with @alexjpwalker here @lolski . We should remove it when we can test it. So that will have to be done manually from client-nodejs when any one of us gets there.